apps/api/src/domain/models/tag/commands/create-tag/create-tag.command-handler.ts

```diff
@@ -46,4 +46,12 @@ export class CreateTagCommandHandler extends BaseCreateCommandHandler<Tag> {
     protected buildEvent(command: CreateTag, eventId: string, userId: string): BaseEvent {
         return new TagCreated(command, eventId, userId);
     }
+
+    protected override async persist(
```

Comment:
Flag this somewhere as a risk of not moving to full event sourcing across the board right now.

Reply:
Better yet, it's been removed.